worlds/sc2/item_names.py

```diff
@@ -458,6 +458,10 @@
 OVERLORD_GENERATE_CREEP                 = "Generate Creep (Overlord)"
 OVERLORD_PNEUMATIZED_CARAPACE           = "Pneumatized Carapace (Overlord)"
 OVERLORD_ANTENNAE                       = "Antennae (Overlord)"
+TYRANNOZOR_BARRAGE_OF_SPIKES            = "Barrage of Spikes (Tyrannozor)"
+TYRANNOZOR_TYRANTS_PROTECTION           = "Tyrant's Protection (Tyrannozor)"
+TYRANNOZOR_HEALING_ADAPTATION             = "Healing Adaptation (Ultralisk/Tyrannozor)"
```

Comment:
Indentation got thrown off by the rename

Reply:
Fixed